boot/boot.go

```diff
@@ -190,6 +190,15 @@ type bootState interface {
 	markSuccessful(bootStateUpdate) (bootStateUpdate, error)
 }
 
+// basicBootState exposes the boot state that is not necessarily tied to a
+// particular snap, but any other type of resource that plays a role in the boot
+// process.
+type basicBootState interface {
```

Comment:
I would name this "successfulBootState"  with a description like   "state of resources that require bookkeeping on a successful boot"